ui/src/components/system/UsersGroups.vue

```diff
@@ -2398,7 +2398,7 @@ export default {
         null,
         function(success) {
           try {
-            success = JSON.parse(success);
+            success = sortList(JSON.parse(success));
```

Comment:
missing `context` here:

`success = context.sortList(JSON.parse(success));`

Reply:
Yes, I saw it :) I will update in a minute. My bad